--- oommf/pkg/oc/oc.tcl	2019-01-28 16:06:22.382192644 +0100
+++ oommf/pkg/oc/oc_patched.tcl	2019-01-29 15:26:14.776390444 +0100
@@ -52,16 +52,16 @@
    }
    return $resultcode
 }
-if {[Oc_AmRoot]>0} {
-   global tcl_platform
-   if {[string compare windows $tcl_platform(platform)]==0} {
-      puts stderr "*** ERROR: Don't run OOMMF as administrator."
-   } else {
-      puts stderr "*** ERROR: Don't run OOMMF as root."
-   }
-   puts stderr "Instead, start OOMMF from a standard user account."
-   exit 666
-}
+# if {[Oc_AmRoot]>0} {
+#    global tcl_platform
+#    if {[string compare windows $tcl_platform(platform)]==0} {
+#       puts stderr "*** ERROR: Don't run OOMMF as administrator."
+#    } else {
+#       puts stderr "*** ERROR: Don't run OOMMF as root."
+#    }
+#    puts stderr "Instead, start OOMMF from a standard user account."
+#    exit 666
+# }
 
 proc Oc_CheckTclIndex {pkg} {
     if {[llength [info commands Oc_DirectPathname]]} {
